### ISSUE TITLE
Crashing when updating to kubejs-forge-1605.3.13-build.50

### ISSUE THREAD

**Issue by Jerome226** [KubeJS-Mods/KubeJS#169]
[latest.log](https://github.com/KubeJS-Mods/KubeJS/files/6911736/latest.log)
Here my latest.log, seems to be caused by a bad mixin or smth.

**Comment by LaughingLeader** [comment]
I have the same issue, but to be specific it's server-side only, when starting a server with only these mods:
```
architectury-1.20.28-forge.jar
kubejs-forge-1605.3.13-build.50.jar
rhino-1605.1.1-build.21.jar
```
Forge:
```
forge-1.16.5-36.2.0.jar
```
Client-side startup works fine with `kubejs-forge-1605.3.13-build.50.jar`.

Additionally, reverting back to `kubejs-forge-1605.3.13-build.45.jar` on the server works.

**Comment by Jerome226** [comment]
Fixed with kubejs-forge-1605.3.13-build.52
